tests/functional/object/rgw/test_obc_quota.py

```
@@ -121,6 +123,457 @@ def test_rgw_obc_quota(
+        except CommandFailed as e:
+            if err_msg in e.args[0]:
+                assert False, "Both quotas were increased but writes still blocked!!"
+            else:
+                logger.error("Copying objects to bucket failed unexpectedly!!")
+                raise
+        else:
+            logger.info("Writes succeeded after patching both quotas!!")
+
+    @tier2
+    @pytest.mark.polarion_id("OCS-8084")
+    def test_rgw_obc_multi_bucket_quota(
```

Ruff (0.16.0)

[warning] Too many branches (17 > 12)

(PLR0912)